manual-deployment.md

```diff
@@ -3,4 +3,164 @@ title: Manual Deployment
+
+## Insecure Cluster
+
+### 1. Set up the first node
+
+Copy the `cockroach` binary to the first machine and then start the node:
+
+~~~ shell
+$ ./cockroach start --insecure --host=node1.example.com
+~~~
+
+This command sets the node to insecure and identifies the address at which other nodes can reach it. Otherwise, it uses all available defaults. For example, the node stores data in the `cockroach-data` directory, listens for internal and client communication on port 26257, and listens for HTTP requests from the Admin UI on port 8080. To set these options manually, see [Start a Node](start-a-node.html). 
+
+### 2. Set up the second node
+
+Copy the `cockroach` binary to the second machine and then start the node:
+    
+~~~ shell
+$ ./cockroach start --insecure --join=node1.example.com:26257
+~~~
+
+The only difference when starting the second node is that you connect it to the cluster with the `--join` flag, which takes the address and port of the first node. Otherwise, it's fine to accept all defaults; since each node is on a unique machine, using identical ports won't cause conflicts.
+
+### 3. Set up additional nodes
+
+Repeat step 2 for each additional node.
+
+### 4. Use the Built-in SQL Client
+
+Start the built-in SQL client from any machine with the `cockroach` binary. This could be one of the node machines or a different machine. 
+
+~~~ shell
+$ ./cockroach sql --insecure --url=postgresql://root@node1.example.com:26257?sslmode=disable
+# Welcome to the cockroach SQL interface.
+# All statements must be terminated by a semicolon.
+# To exit: CTRL + D.
+~~~
+
+This command uses the `--url` flag to identify the client user and the hostname and port of the node to connect to. You can also specify these details with distinct flags. For more information, see [Use the Built-In SQL Client](use-the-built-in-sql-client.html).
+
+Once you're connected, run some [SQL statements](learn-cockroachdb-sql.html): 
+
+~~~ shell
+root@:26257> CREATE DATABASE bank;
+CREATE DATABASE
+
+root@:26257> SET DATABASE = bank;
+SET DATABASE
+
+root@:26257> CREATE TABLE accounts (id INT PRIMARY KEY, balance DECIMAL);
+CREATE TABLE
+
+root@26257> INSERT INTO accounts VALUES (1234, DECIMAL '10000');
+INSERT 1
+
+root@26257> SELECT * FROM accounts;
++------+---------+
+|  id  | balance |
++------+---------+
+| 1234 |   10000 |
++------+---------+
+~~~
+
+## Secure Cluster
+
+### 1. Create security certificates
+
+On a machine that won't be part of the cluster, create the CA, node, and client certificates and keys:
+
+~~~ shell
+# Create the CA certificate and key.
+$ ./cockroach cert create-ca --ca-cert=ca.cert --ca-key=ca.key
+   
+# Create the certificate and key for nodes.
+# Be sure to include the hostnames of each machine in the cluster.
+$ ./cockroach cert create-node node1.example.com node2.example.com node3.example.com --ca-cert=ca.cert --cert=node.cert --key=node.cert
```

Comment:
@mberhault, even after our long talk this morning, I think I got this wrong again. On this page, I'm saying to create one node cert with the hostnames of every node. Based on our talk with Ben this morning, I should recommend creating a unique certificate for each node, each time including just the specific node's hostnames (not hostnames of other nodes), is that right?